lib/etl/saver.rb

```diff
@@ -0,0 +1,32 @@
+require 'etl/errors'
+
+module Etl
+  class Saver
```

Comment:
I would maybe call this a `Loader` since we have adopted the terms Extract, Transform, Load.